packages/react/src/hooks/useRemoteParticipant.ts

```diff
@@ -35,9 +35,14 @@ export function useRemoteParticipant(
     () => connectedParticipantObserver(room, identity, { additionalEvents: updateOnlyOn }),
     [room, identity, updateOnlyOn],
   );
-  const participant = useObservableState(
-    observable,
-    room.getParticipantByIdentity(identity) as RemoteParticipant | undefined,
-  );
-  return participant;
+
+  const [participantWrapper, setParticipantWrapper] = React.useState({
```

Comment:
Could you add a comment outlining why this is needed? It'll make it easier to understand in the future.

Reply:
Done, is it understandable?